### PR TITLE
Add test

### PR DESCRIPTION
**Pull request by minhphambocasay**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "build": "umi build",
     "lint": "eslint --ext .js src mock tests && npm run lint:style && npm run lint:prettier",
     "lint-staged": "lint-staged",
-    "lint-staged:js": "eslint --ext .js",
+    "lint-staged:js": "eslint --ext .js --fix",
```
Comment (ThangNguyen-Bocasay) [inline package.json]: Please remove --fix option

File: package.json [hunk 2]
```diff
@@ -42,6 +42,7 @@
   "dependencies": {
     "@antv/data-set": "^0.10.1",
     "antd": "^3.16.1",
+    "axios": "^0.18.0",
```
Comment (ThangNguyen-Bocasay) [inline package.json]: Please use default request instead

File: src/pages/Note/models/noteModel.js [hunk 3]
```diff
@@ -1,24 +1,100 @@
...
+        yield put({
+          type: 'DELETE_NOTE_ERROR',
+          payload: {},
+        });
+      }
     },
   },
   reducers: {
-    loadedNotes(state, { payload }) {
+    GET_ALL_NOTES_SUCCESS(state, { payload }) {
```
Comment (ThangNguyen-Bocasay) [inline src/pages/Note/models/noteModel.js]: Change function name to lowercase

File: src/pages/NoteItem/index.js [hunk 4]
```diff
@@ -0,0 +1,77 @@
+/* eslint-disable no-prototype-builtins */
+/* eslint-disable no-param-reassign */
+import React from 'react';
+import { connect } from 'dva';
+
+const NoteItem = ({ handleDeleteNote, dispatch, handleUpdateNote, item }) => {
+  // eslint-disable-next-line no-param-reassign
+  handleDeleteNote = data => {
```
Comment (ThangNguyen-Bocasay) [inline src/pages/NoteItem/index.js]: Should use useCallback at here

File: src/pages/NoteItem/index.js [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+    dispatch({
+      type: 'noteModel/DELETE_NOTE',
+      payload: data,
+    });
+  };
+
+  handleUpdateNote = (data, index, e) => {
```
Comment (ThangNguyen-Bocasay) [inline src/pages/NoteItem/index.js]: Should use useCallback at here

File: src/pages/NoteItem/index.js [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+        checked={item.is_doing}
+        disabled={item.is_done}
+        onChange={() => handleUpdateNote(item, 'is_doing')}
+      />
+    </div>
+  );
+};
+
+export default connect(models => {
+  console.log(models);
```
Comment (ThangNguyen-Bocasay) [inline src/pages/NoteItem/index.js]: remove console.log

File: src/pages/NoteItem/index.js [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+  // if (models.noteModel.udpatedNote && models.noteModel.udpatedNote.id === ite) {
+  // return { item: models.noteModel.udpatedNote }
+  // }
+  return {};
```
Comment (ThangNguyen-Bocasay) [inline src/pages/NoteItem/index.js]: If it returns nothing, pls write like this:
connect(() => ({}))(NoteItem)

**Review by ThangNguyen-Bocasay — APPROVED** [review]

**Review by anhdaobocasay — APPROVED** [review]

**Pull request merged**